src/components/content/exercise.tsx

```diff
@@ -31,33 +67,40 @@ export function Exercise(props: ExerciseProps) {
   } = props
   const [solutionVisible, setVisible] = React.useState(false)
 
-  let taskValue = task.children
-  let solutionValue = solution.children
+  let taskValue = {}
+  let solutionValue: unknown[] = []
```

Comment:
is this even used anymore? Saw you referring to `solution.children` directly in the code below.